src/textual/css/query.py

```diff
@@ -36,14 +37,16 @@ def __init__(
         self,
         node: DOMNode | None = None,
         selector: str | None = None,
-        nodes: list[DOMNode] | None = None,
+        nodes: list[Widget] | None = None,
```

Comment:
Queries now return Widgets rather than DOMNodes. This means that it excludes App, which is find since it is a special object that doesn't have any visuals. Although you can still reference App in a selector.